tests/e2e/cypress/integration/spanish.spec.js

```diff
@@ -0,0 +1,49 @@
+describe("Index page - Internationalization spec", () => {
+  it("Gives user Spanish language option", () => {
+    cy.visit("/")
+
+    cy.get("#header").should("contain", "Español")
+    cy.get("#header").should("not.contain", "English")
+  })
+
+  it("Changes the language to Spanish", () => {
+    cy.visit("/")
+
+    cy.contains("Español")
+      .click()
+
+    cy.contains("La nueva forma de pagar")
+    cy.get("#header").should("not.contain", "Español")
+    cy.get("#header").should("contain", "English")
+  })
+
+  it("Changes the language from Spanish back to English", () => {
+    cy.visit("/")
+
+    cy.contains("Español")
+      .click()
+
+    cy.contains("La nueva forma de pagar")
+    cy.get("#header").should("not.contain", "Español")
+    cy.get("#header").should("contain", "English")
+    cy.contains("English")
+      .click()
+
+    cy.get("#main-content").should("contain", "The new way to pay for transit")
+    cy.get("#main-content").should("not.contain", "La nueva forma de pagar")
```

Comment:
Check the button again after changing back

```js
cy.get("#header").should("contain", "Español")
cy.get("#header").should("not.contain", "English")
```

Reply:
To be resolved in different PR.